src/components/views/rooms/RoomPreviewBar.js

```diff
@@ -492,9 +492,13 @@ export default class RoomPreviewBar extends React.Component {
                 }
 
                 const myUserId = MatrixClientPeg.get().getUserId();
-                const reason = this.props.room.currentState.getMember(myUserId).events.member.event.content.reason;
-                if (reason) {
-                    reasonElement = <InviteReason reason={reason} />;
+                const memberEventContent = this.props.room.currentState.getMember(myUserId).events.member.event.content;
+
+                if (memberEventContent.reason) {
+                    reasonElement = <InviteReason
+                        reason={memberEventContent.reason}
+                        htmlReason={memberEventContent["io.element.html_reason"]}
```

Comment:
I think I would feel a bit better if `io.element.html_reason` were to be a `const` somewhere, though I don't know where exactly. Perhaps someone from the team will have an idea

Reply:
It is a `const` on module level now. If there is a better place to put it please let me know.